src/net_processing.cpp

```diff
@@ -2221,28 +2222,26 @@ void PeerManagerImpl::SendBlockTransactions(CNode& pfrom, Peer& peer, const CBlo
  *
  * We'll send a getheaders message in response to try to connect the chain.
  *
- * The peer can send up to MAX_UNCONNECTING_HEADERS in a row that
+ * The peer can send up to MAX_UNCONNECTING_HEADERS_MSGS in a row that
```

Comment:
Care to address [this comment ](https://github.com/bitcoin/bitcoin/pull/25454/files/99f4785cad94657dcf349d00fdd6f1d44cac9bb0#r912929185) while you're touching this line?

Reply:
Thanks, done.